### PR TITLE
Dev 49 - added max-height to all pages

### PR DESCRIPTION
**Pull request by KovDimaY**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure all the tests are still passing.
* [x] Make sure the test coverage is still 100%.
* [x] Make sure your source branch is correct.
* [x] Make sure your destination branch is **devel**.
* [x] Make sure the name of the PR has a correct DEV-# identification prefix.
* [ ] Make sure you reviewed all your changes and they look correct.
* [x] Make sure all the commits have a correct format. 
* [x] Make sure you refer related issues for you PR. 
* [x] Make sure you add **KovDimaY** as a reviewer and assignee of your PR.

Take a look at the Issue #92

### CODE REVIEW

File: client/src/containers/EditUserProfile/styles.css [hunk 1]
```diff
@@ -63,6 +65,8 @@
     flex-direction: row;
```
Comment (NikolayYakovenko) [inline client/src/containers/EditUserProfile/styles.css]: This (and line below) will works if you set `display: flex`

Comment (KovDimaY) [reply client/src/containers/EditUserProfile/styles.css]: Yes, exactly, it is written just the line before... 😉 😄

**Comment by NikolayYakovenko** [comment]
Good job!

**Comment by NikolayYakovenko** [comment]
Good job!

**Pull request merged**